pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/Functions.java

```diff
@@ -259,4 +260,11 @@
      * @throws PulsarAdminException 
      */
     List<WorkerInfo> getCluster() throws PulsarAdminException;
+    
+    /**
+     * Get function worker metrics
+     * @return
+     * @throws PulsarAdminException 
+     */
+    Metrics getMetrics() throws PulsarAdminException;
```

Comment:
Then maybe we change this to getWorkerMetrics?

Reply:
I'm still a bit confused here. Are you getting worker related metrics or function related metrics?

Reply:
same metrics which is being exposed on `/metrics` endpoint for prometheus but it will be in json format. it will be used by our non-prometheus monitoring system to aggregate all functions-metrics owned  by the worker.

Reply:
I see a disconnect here. Most of other calls listed here are all function specific that get routed appropriately to the right worker who then answers with the right information.
On the other hand, this particular call is for getting the metrics housed by a specific worker. Do we want a different place to house such calls?

Reply:
> On the other hand, this particular call is for getting the metrics housed by a specific worker. Do we want a different place to house such calls?

hmm.. yes. right now, in broker we have separate resource [BrokerStats](https://github.com/apache/incubator-pulsar/blob/master/pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokerStatsBase.java) to serve broker-localhost stats, probably we can add resource: `WorkerStats` for consistency and we can add worker-specific stats in the same resource as needed.

Reply:
changed it and added separate endpoint.